tests/test_generators/test_docgen.py

```diff
@@ -526,21 +554,88 @@ def test_docgen_multiline_everything(
             + f"| [{class_name}]({class_name}.md) | {description} |  no  |\n"
         ) in slot_file_contents
 
+
+@pytest.mark.parametrize("slot", UC_NAMES)
+@pytest.mark.parametrize("truncate_descriptions", [True, False])
+@pytest.mark.parametrize("hierarchical_class_view", [True, False])
+def test_docgen_multiline_everything_class_page(
+    tmp_path: Generator[Path, None, None],
+    multi_line_description_test_schema: str,
+    hierarchical_class_view: bool,
+    truncate_descriptions: bool,
+    slot: str,
+) -> None:
+    """Tests that single and multi-line descriptions are rendered correctly with truncation enabled and disabled.
+
+    :param tmp_path: temp dir for doc gen output
+    :type tmp_path: Generator[Path, None, None]
+    :param multi_line_description_test_schema: schema with lots of descriptions
+    :type multi_line_description_test_schema: str
+    :param hierarchical_class_view: whether or not hierarchical class view is enabled
+    :type hierarchical_class_view: bool
+    :param truncate_descriptions: whether or not description truncation is enabled
+    :type truncate_descriptions: bool
+    :param slot: the slot being tested in this particular test
+    :type slot: str
+    """
+    gen = DocGenerator(
+        multi_line_description_test_schema,
+        hierarchical_class_view=hierarchical_class_view,
+        truncate_descriptions=truncate_descriptions,
+        mergeimports=False,
+    )
+    gen.serialize(directory=str(tmp_path))
+
     # check the table in the populated class file, ClassMultiLinePreserveWhitespace.md
     class_file = tmp_path / f"Class{UC_NAMES[MULTI_LINE_WS]}.md"
     with class_file.open() as fh:
         class_file_contents = fh.read()
 
-        for slot in UC_NAMES:
-            description = ""
-            if slot != NO_DESC:
-                description = DESCRIPTIONS.get(slot)[0] if truncate_descriptions else DESCRIPTIONS_MD.get(slot)
-            assert (
-                f"| [slot_{slot}](slot_{slot}.md) |"
-                # all slots are type string
-                + " 0..1 <br/> [xsd:string](http://www.w3.org/2001/XMLSchema#string) "
-                + f"| {description} | direct |"
-            ) in class_file_contents
+        class_file_list = [line for line in class_file_contents.split("\n") if ".md" in line]
+        print(class_file_list)
+
+        slot_type = "[xsd:string](http://www.w3.org/2001/XMLSchema#string)"
+        if WINDOWS:
+            slot_type = "[xsd:string](xsd:string)"
```

Comment:
@ialarmedalien - Do you know why the CURIE is not replaced by the URI here? I have seen the same when I saw you struggling with Windows but could not yet understand why it happens.

My comment should not hold this PR. Fixing the issues that break the tests on main is too important.

Reply:
There is something going on with the link generation function (`link(...)` in the DocGenerator package) -- I don't have a windows box to test on so I didn't want to delay getting things fixed by investigating further. I'm wondering if it's a path issue around accessing the local metamodel.